src/views/product-downloads-view/components/downloads-section/toast-on-copy.tsx

```diff
@@ -0,0 +1,40 @@
+import { toast, ToastColor } from 'components/toast'
+import { IconCheckCircle24 } from '@hashicorp/flight-icons/svg-react/check-circle-24'
+import { IconAlertTriangle24 } from '@hashicorp/flight-icons/svg-react/alert-triangle-24'
+
+/**
+ * Given a copySuccess state,
+ * as well as an OS and optional package manager name,
+ *
+ * Display a "successful copy" toast if the copySuccess state is true,
+ * or a "failed to copy" toast if copySuccess is false.
+ *
+ * If copySuccess is "null" (as it is in its initial state),
+ * this is fine, but it does not merit toast.
+ */
+export default function toastOnCopy(
```

Comment:
`toastOnCopy`, as its name promises, displays a toast notification when the `copySuccess` state changes to either `true` (successful copy!) or `false` (failed copy).